src/utils/workoutSessionMirroringStartHandler.ts

```diff
@@ -0,0 +1,5 @@
+import Native from '../native-types'
+
+const workoutSessionMirroringStartHandler = async () => Native.workoutSessionMirroringStartHandler()
```

Comment:
Do we want to put any return types on this? 
```suggestion
const workoutSessionMirroringStartHandler: Promise<void> = async () => Native.workoutSessionMirroringStartHandler()
```

Reply:
Does the function return anything?

Reply:
The return type of this is: `() => Promise<boolean>`. Added it!